### ISSUE TITLE
MalformedURLException When Using Browser Proxying

### ISSUE THREAD

**Issue by ryanjbaxter** [wiremock/wiremock#1948]
I am running into the following exception when using Wiremock with browser proxying enabled using Wiremock `2.33.2` and Jetty `9.4.48.v20220622`.

```
[qtp1413378318-32] WARN org.eclipse.jetty.server.HttpChannel - handleException null java.net.MalformedURLException: For input string: "59141null"
Exception in thread "main" org.apache.hc.client5.http.ClientProtocolException: CONNECT refused by proxy: HTTP/1.1 500 Server Error
	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:173)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:75)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:89)
	at org.example.App.main(App.java:72)
Caused by: org.apache.hc.client5.http.impl.TunnelRefusedException: CONNECT refused by proxy: HTTP/1.1 500 Server Error
	at org.apache.hc.client5.http.impl.classic.ConnectExec.createTunnelToTarget(ConnectExec.java:259)
	at org.apache.hc.client5.http.impl.classic.ConnectExec.execute(ConnectExec.java:145)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
	at org.apache.hc.client5.http.impl.classic.ProtocolExec.execute(ProtocolExec.java:175)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
	at org.apache.hc.client5.http.impl.classic.HttpRequestRetryExec.execute(HttpRequestRetryExec.java:96)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
	at org.apache.hc.client5.http.impl.classic.ContentCompressionExec.execute(ContentCompressionExec.java:133)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement$1.proceed(ExecChainElement.java:57)
	at org.apache.hc.client5.http.impl.classic.RedirectExec.execute(RedirectExec.java:115)
	at org.apache.hc.client5.http.impl.classic.ExecChainElement.execute(ExecChainElement.java:51)
	at org.apache.hc.client5.http.impl.classic.InternalHttpClient.doExecute(InternalHttpClient.java:170)
	... 3 more
```
I am not sure if this a problem with Jetty or if there needs to be some changes in Wiremock to work with this version of Jetty.  If I use Jetty `9.4.46.v20220331` everything works fine.  There appears to be some changes in the way Jetty is handling CONNECT requests and I assume that might be the root cause.

https://github.com/eclipse/jetty.project/commit/4ca8afbbd667c19a084d5ff14a4f08eb7049d1c7#diff-e6545ae2dcb35a2122c554fedb35dbe62f7f26750501e373d7a9a9c776dd1deaL311

I have attached a sample project that reproduces the issue.
[test-wiremock.zip](https://github.com/wiremock/wiremock/files/9501518/test-wiremock.zip)

**Comment by ryanjbaxter** [comment]
If I use the standalone jar I can work around this problem, but I will leave this issue open for future reference because it will be an issue once the version of Jetty is updated

**Comment by tomakehurst** [comment]
This is fixed in bbbc03f5a0ebcf2377cc5e33e2c067214dc1b06b and will be released shortly in 2.34.0
